### PR TITLE
review please

### PR DESCRIPTION
**Pull request by khsishk**
review please

### CODE REVIEW

**Comment by Ji-ho-Kim** [comment]
i think its fine

**Review by Ji-ho-Kim — APPROVED** [review]
try reviewing

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # skku-11
 # jhk
 # hs
+fork by hs
+
```
Comment (Ji-ho-Kim) [inline README.md:5]: fork by jh

**Review by Ji-ho-Kim — APPROVED** [review]
review2

**Review by 02hyeok — APPROVED** [review]
👍

**Review by Ji-ho-Kim — APPROVED** [review]
final review

**Pull request merged**